src/commit.c

```diff
@@ -80,8 +80,8 @@ static int git_commit__create_buffer_internal(
 }
 
 static int validate_tree_and_parents(git_array_oid_t *parents, git_repository *repo, const git_oid *tree,
-				     git_commit_parent_callback parent_cb, void *parent_payload,
-				     const git_oid *current_id, bool validate)
+						 git_commit_parent_callback parent_cb, void *parent_payload,
```

Comment:
(There's something going on indent-wise in those 3 hunks).

Reply:
No kidding! I wonder what happened there.